oemofB3/model_structure/busses.csv

```diff
@@ -1,5 +1,6 @@
 carrier,balanced,description
 electricity,true,Electricity bus
-heat,true,Heat bus
-ch4,false,CH4 bus
-uranium,false,Uranium bus
\ No newline at end of file
+heat,false,Heat bus
```

Comment:
Would be more consistent to make all buses balanced and include a heat-shortage.

Reply:
Ah, thats a mistake. I remove the heat bus because I don't have one. I thougth, the 'False' stands for existing or not.

Reply:
I see! You could remove it, but this would affect all scenarios.

Reply:
I made the heat bus balanced again in 70c56c3 and did not remove it, as it will be part of the model later. The resource busses have to be unbalanced because there are no commodity sources defined. There is no way to exclude unused buses at the moment depending on the scenario. I opened an issue (#18)